metaproc-connector-demo/config/variables.yaml

```diff
@@ -3,4 +3,4 @@ Variables:
     Url: http://localhost:8080
     Username: admin
     Password: 1234
-    TokenExpiredDuration: 3600000 # Milisecond
+    TokenExpiredDuration: 60 # Minute
```

Comment:
We should add a comment above the variable according to Reguel's suggestion.
"# Duration in minutes
TokenExpiredDuration: 60"